ndc-lambda-sdk/test/execution/execute-query.test.ts

```diff
@@ -289,7 +289,7 @@ describe("execute query", function() {
       },
       objectTypes: {},
       scalarTypes: {
-        "String": {}
+        "String": { type: "built-in" }
```

Comment:
Yeah let's have a constant for sure.

Reply:
Seems hardly worth it, given its not a magical value (obvious what it is), and that the most likely predicted change to this (adding a property to that object) would require changing in all locations anyway (to add the specific value of that property) and therefore the removal of the constant.

I'm not gonna die on this hill though, so if you feel strongly about it, I'll extract all instances to a constant. (Keep in mind `const builtIn = { type: "build-in" }` is not a true constant, it's a potentially mutable shared instance...)

Reply:
Related comment on ndc spec update PR: https://github.com/hasura/ndc-nodejs-lambda/pull/8#discussion_r1491861932

Reply:
Ok well feel free to keep it if you prefer.